packages/opentelemetry-instrumentation-watsonx/tests/conftest.py

```diff
@@ -20,7 +20,14 @@ def exporter():
     provider.add_span_processor(processor)
     trace.set_tracer_provider(provider)
 
-    WatsonxInstrumentor().instrument()
+    try:
+        # Check for required package in the env, skip test if could not found
+        from ibm_watsonx_ai.foundation_models import ModelInference
```

Comment:
don't we need to add them to poetry otherwise they won't get installed in the test?

Reply:
@nirga I'm not sure if we add the `ibm-watsonx-ai` import in poetry, would it cause issue for test on 3.8 and 3.9? is there a way to set it to only run test on 3.10+?

Reply:
@nirga test execution in CI looks good to me. It skips watsonx test on Python < 3.10, but runs normally otherwise. Traceloop SDK tests are running for all versions.
I think this is a good solution.

Reply:
@paolorechia but how does it run if the Watson package is not installed? 🤔

Reply:
Aha! It's only running the try/catch, and exiting on the except then? I missed that.

Reply:
Yep 😅

Reply:
OK, but at least the @huang-cn is able to run tests locally now. 

Do you think we might need to revive the optional dependencies issue I created but closed? Maybe it's a way forward for these issues. I don't think it's reasonable to request the PR author to implement a fix for this though.

Reply:
Yeah let's merge it, open an issue and figure out if there's a way around it within poetry